driftbase/tests/players/test_players.py

```diff
@@ -91,6 +91,18 @@ def test_players(self):
         self.get(self.endpoints["players"] + "/{}".format(big_number),
                  expected_status_code=http_client.NOT_FOUND)
 
+    def test_find_player(self):
+        self.auth()
+        player_url = self.endpoints["my_player"]
+        player_name = "Spicy Meatball"
+        self.patch(player_url, data={"name": player_name})
+        for search_string in (player_name, "*icy*"): # Test both exact and fuzzy searches

```

Comment:
Do these matches ignore case? The test doesn't account for that.

Reply:
The test ought to add players that don't match the search as well, to ensure they aren't returned, as well as test for multiple results.

Reply:
wrt case-sensitivity, I just decided (for this implementation) that this would support 'exact' search, which is case sensitive and 'fuzzy' which is not case-sensitive and allows wildcards